lbz/exceptions.py

```diff
@@ -26,12 +26,13 @@ class LambdaFWException(Exception):
     status_code = HTTPStatus.INTERNAL_SERVER_ERROR.value
     error_code: str | None = None
 
-    def __init__(self, message: str = "", error_code: str | None = None) -> None:
-        super().__init__(message)
-        if message:
-            self.message = message
-        if error_code:
-            self.error_code = error_code
+    def __init__(
+        self, message: str = "", error_code: str | None = None, extra: dict | None = None
+    ) -> None:
+        self.message = message or self.message
+        super().__init__(self.message)
```

Comment:
There should be nothing before super initialization - inheritance is based on extending the base class 😉 
```suggestion
        super().__init__(message)

        self.message = message or self.message
```